### DISCUSSION TITLE
Space-specific application and folder buttons

### DISCUSSION THREAD

**Discussion by producation** [lawand-dot-io/taskbar#18] [Feature requests]
In addition to see just the windows from the current Space it would be great if you could have like Spaces Presets. Like I have a Space for Coding where only all of my Coding Apps are pinned and another Space for Social Media where just my Social Media Apps are pinned

**Comment by lawand-dot-io** [comment]
Hello. Can you explain this more? I personally also usually have different spaces for different purposes. This is already possible in Taskbar as it shows the current space's windows only, so you can open those windows in each space and get the result you're asking for. Perhaps what you're saying is the ability to automatically open all those windows in the right space? Or do you mean assigning apps for specific spaces so that when you launch them they launch in a specific space?

**Reply by producation** [reply]
It's just about the pinned apps. Right now when I pin an app it is pinned global on every space so it gets messy real quick. 
It would save a lot of time when I open a new Space that I can e.g. right click on the taskbar and choose from a set of workspaces where different apps and folders are pinned. 
When I start coding I have a few folders which I mostly open and different programs from which I choose. When those apps and folders could be pinned to a specific taskbar I don't need to build my own set of shortcut folders to start them since Launchpad is a mess to maintain and my Application folder has like 200+ apps.

**Reply by lawand-dot-io** [reply]
Ah, I see what you mean. Yeah, could be a good idea. Will consider it for a future version. Also pinning specific folders. Cheers :)

**Reply by lawand-dot-io** [reply]
Removed the part of the comment about adding Finder buttons that launch specific folders as that's already included in https://github.com/lawand-dot-io/taskbar/discussions/12
